ILSANG/Sources/Views/Setting/CustomerServiceView.swift

```diff
@@ -16,23 +17,47 @@ struct CustomerServiceView: View {
                 dismiss()
             }
             
-            List {
-                NavigationLink {
-                    //
-                } label: {
-                    Text("자주 묻는 질문")
-                }
+            HStack {
+                Text("인스타그램")
+                    .font(.system(size: 17).bold())
+                    .foregroundColor(.gray400)
                 
-                Text("이메일주소: findtastyquest@gmail.com")
+                Spacer()
+                
+                Text("1sang.official")
+                    .underline()
+                    .font(.system(size: 17))
+                    .foregroundColor(.gray200)
+            }
+            .padding(.horizontal, 25)
+            .padding(.vertical, 26)
+            .background(.white)
+            .onTapGesture {
+                openInsragram()
             }
-            .listRowBackground(Color.white)
-            .listStyle(.plain)
+            
+            Spacer()
+            
         }
         .frame(maxWidth: .infinity)
+        //MARK: 색상 변경시 수정
+        .background(Color.gray100)
```

Comment:
```suggestion
        .background(Color.background)
```

배경 컬러 에셋에 추가되어있습니다!-!

Reply:
두 가지 수정 완료했습니다!

Reply:
@HIBAR97  확인했습니다!